src/SBMS/sbms.py

```diff
@@ -1194,7 +1195,7 @@ def AddEvtGen(env):
 	else:
                 AddHepMC(env)
                 AddPhotos(env)
-                EVTGEN_CPPPATH = "%s/" % (EVTGEN_HOME)
+                EVTGEN_CPPPATH = "%s/include" % (EVTGEN_HOME)
```

Comment:
This breaks the convention in the build_scripts environment and is responsible for the failed test